### PR TITLE
Update GIBS intro text for WBC version

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Update the GIBS intro text that appears on [preview.va.gov](https://preview.va.gov/education/gi-bill/post-9-11/ch-33-benefit/)

## Testing done
Local testing

## Screenshots
**Before**
image

**After**
image

## Acceptance criteria
- [ ] 

## Definition of done
~~- [ ] Events are logged appropriately~~
~~- [ ] Documentation has been updated, if applicable~~
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by erikphansen** [comment]
@peggygannon Once merged this won't roll out to preview.va.gov until the daily deploy to prod which takes place 3PM EST.

**Review by peggygannon — APPROVED** [review]
Hi, @erikphansen Looks good to me. thanks!

File: src/applications/post-911-gib-status/containers/IntroPageSummary.jsx [hunk 1]
```diff
@@ -61,12 +61,15 @@ export default function BrandConsolidationSummary() {
...
-          find out how much money you have left to use for your education or
-          training.
+          If you were awarded Post-9/11 GI Bill education benefits, your GI Bill
+          Statement of Benefits will show you how much of your benefits you’ve
+          used and how much you have left to use for your education or training.
+          These education benefits can help cover some or all of the costs for
+          school or training.
+        </p>
+        <p>
+          You'll be able to view this benefit statement only if you were awarded
```
Comment (cehsu) [inline src/applications/post-911-gib-status/containers/IntroPageSummary.jsx]:
```suggestion
          You’ll be able to view this benefit statement only if you were awarded
```

**Review by cehsu — APPROVED** [review]
Looks good.

**Pull request merged**